core/modules/qproc/QuerySession.cc

```diff
@@ -402,28 +394,32 @@ std::vector<std::string> QuerySession::_buildChunkQueries(ChunkSpec const& s) co
             std::stringstream ss;
             std::copy(sVector.begin(), sVector.end(),
                       std::ostream_iterator<ChunkSpecSingle>(ss, ","));
-            LOGF_INFO("subchunks: %1%" % ss.str());
+            LOGF(getLogger(), LOG_LVL_INFO, "subchunks: %1%" % ss.str());
```

Comment:
Can you remove stream above and replace it with `printable(sVector)`? `if (LOG_CHECK_INFO())` will not be needed then?